examples/web-nextjs/README.md

```diff
@@ -0,0 +1,38 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Pls update this README file to explain that this is plain NextJS app straight from the CLI tool with Reassure atteched. And how to run the reassure tests.